src/crystal/system/system.cr

```diff
@@ -0,0 +1,24 @@
+# :nodoc
+module Crystal
+  # :nodoc
+  module System
+    # :nodoc
+    module System
```

Comment:
@bcardiff If we changed that name we'd have to change the name of `System` to `Inspect` which would make little sense. I'd rather have bit of duplication than a nonsensical name.